generator/src/main/java/com/scottlogic/deg/generator/CommandLine/CommandLineValidator.java

```diff
@@ -0,0 +1,33 @@
+package com.scottlogic.deg.generator.CommandLine;
+
+import com.scottlogic.deg.generator.generation.GenerationConfig;
+
+import java.util.ArrayList;
+
+/**
+ * Class used to determine whether the command line options are valid for generation
+ * */
+public class CommandLineValidator {
+
+    public ValidationResult validateCommandLine(GenerationConfig config) {
+
+        ArrayList<String> errorMessages = new ArrayList<>();
+        ValidationResult validationResult = new ValidationResult(errorMessages);
+
+        if (config.getDataGenerationType() == GenerationConfig.DataGenerationType.RANDOM
+            && !config.getMaxRows().isPresent()) {
+
+            errorMessages.add("RANDOM mode requires max row limit\nuse -n=<row limit> option\n");
```

Comment:
A new line shouldn't be added here, thats the responsibility of whatever presents this error

Reply:
Ok, decided to create a class to deal with the printing and formatting of the error messages